ot/da.py

```diff
@@ -843,10 +927,896 @@ def predict(self, x):
 
         if self.computed:
             K = kernel(
-                x, self.xs, method=self.kernel, sigma=self.sigma, **self.kwargs)
+                x, self.xs, method=self.kernel, sigma=self.sigma,
+                **self.kwargs)
             if self.bias:
                 K = np.hstack((K, np.ones((x.shape[0], 1))))
             return K.dot(self.L)
         else:
             print("Warning, model not fitted yet, returning None")
             return None
+
+##############################################################################
+# proposal
```

Comment:
remove proposal, we are near convergence here

Reply:
Ok good point